star-ros2/src/star_perception/star_perception/_hailo_worker.py

```
@@ -0,0 +1,196 @@
+        per_class = list(arr)
+
+    out = []
+    for class_id, dets in enumerate(per_class):
+        arr = np.asarray(dets)
+        if arr.size == 0 or arr.ndim != 2 or arr.shape[1] < 5:
+            continue
+        for row in arr:
+            score = float(row[4])
+            if score <= 0.0:
+                continue
+            y_min = float(row[0]); x_min = float(row[1])
```

Ruff (0.15.10)

[error] Multiple statements on one line (semicolon)

(E702)